src/status_im2/contexts/wallet/events.cljs

```diff
@@ -17,14 +17,15 @@
 
 (rf/reg-event-fx :wallet/show-account-created-toast
  (fn [{:keys [db]} [address]]
-   (let [{:keys [name]} (get-in db [:wallet :accounts address])]
+   (let [account (get-in db [:wallet :accounts address])]
      {:db (update db :wallet dissoc :navigate-to-account :new-account?)
-      :fx [[:dispatch
-            [:toasts/upsert
-             {:id         :new-wallet-account-created
-              :icon       :i/correct
-              :icon-color colors/success-50
-              :text       (i18n/label :t/account-created {:name name})}]]]})))
+      :fx [(when account
```

Comment:
some strange issue where it's not finding the "name" for watched accounts but is for normal accounts 🤷‍♂️ .
If I log `get-in db [:wallet :accounts])` the corresponding account details are there but the look up is not working by address.
Quite strange so I might create a separate issue to investigate it and not block watch only accounts functionality.

Reply:
Maybe something related to `clojure.core/name` :thinking: 
If you need help investigating the issue, let me know :+1:

Reply:
thanks @ulisesmac - given that watch-only accounts are helpful for testing data (e.g we can use accounts with collectibles/assets etc) I will not block this pr but add it as a follow up issue. The address is added find and there's no issues. The name showed once or twice for me, but mostly not. but you could be right with the shadowed var, although I logged the data and it was not showing at all in some cases so I think some form of timing issue, pero no sé

Reply:
if we have an open issue for this, I can take a look :+1:

Reply:
Ok nice, I will create it!

Reply:
@ulisesmac found the problem :) The problem was that when you copy an address from Etherscan the letters are uppercase however Status go always returns these as lower case. So then for watch only accounts it could not find the account in the map. 
cc @smohamedjavid 

I think we should eventually add some handling to the address input page so it goes lower case, but for now I will force the event handling the add account to lowercase 👍